action_server: 'RCoM_ActionServer'
control_client: 'PositionJointInterface_trajectory_controller/follow_joint_trajectory'
planning_group: 'arm_endoscope'

# task and remote center of motion gains - proportional, integral, differential
kpt: [10.0,10.0,10.0]
kit: [0.0,0.0,0.0]
kdt: [0.0,0.0,0.0]
kprcm: [40.0, 40.0, 40.0]
kircm: [0.0, 0.0, 0.0]
kdrcm: [0.0, 0.0, 0.0]
lambda0: 0.4

# control interval, in velocity control: rate at which rcom goals are sent, else: sets interval to execute incremental motion
dt: 0.002

# velocity scaling
alpha: 0.1

# link names
link_pi: 'storz_endoscope_link_1'
link_pip1: 'storz_endoscope_link_cm'

# threshold 1, divergence check prior to execution
t1_td: 0.1
t1_p_trocar: 0.1

# threshold 2, convergence check after execution
t2_td: 0.0001
t2_p_trocar: 0.0001

# threshold scale
t_td_scale: [1.0, 1.0, 1.0]

# max number of iterations
max_iter: 20
